examples/expr2.py

```diff
@@ -1,10 +1,6 @@
-def main0():
-    x: i32
-    x = (2+3)*5
+def main():
+    x: symbolic = Symbolic('x')
     print(x)
 
-main0()
+main()
```

Comment:
Currently, this doesn't work in CPython. I think we handle this ltypes and make sure this works with CPython